zipline/assets/asset_writer.py

```diff
@@ -102,6 +104,16 @@
     'exchange': None,
 }
 
+# Default values for the supplementary_mappings DataFrame
+_supplementary_mappings_defaults = {
+    'sid': None,
+    'value': None,
+    'field': None,
+    'start_date': 0,
+    'end_date': 2 ** 62 - 1,
```

Comment:
Traced this back to https://github.com/quantopian/zipline/commit/ee9c79a6f849e4d7e9035bc58977bce125ef6f5e. There's no indication what the problem with the actual max int64 was, but this might not be an issue anymore since we no longer support 3.3.